sdk/Baggage.php

```diff
@@ -4,23 +4,20 @@
 
 namespace OpenTelemetry\Sdk;
 
+use Generator;
 use OpenTelemetry\Context\Context;
 use OpenTelemetry\Context\ContextKey;
 
-/**
- * @extends Context<Baggage>
- */
 class Baggage extends Context
```

Comment:
Ultimately I think it would make more sense to better follow Java's implementation and make `Baggage` _not_ extend `Context`, but instead implement `ImplicitContextKeyed` itself.

That'll be for another PR tho.

Reply:
Sounds good to me.  Maybe worth making an issue about it?

Reply:
Was planning on tackling Baggage after getting context/propagation up to date. But I can create a more generic Baggage issue if you want it for tracking?